exporter/views.py

```diff
@@ -35,13 +37,81 @@ def get_original_asset_id(download_url):
     """
     if download_url.startswith("http://tile.loc.gov/"):
         pattern = r"/service:([A-Za-z0-9:\-]*)/"
-        asset_id = re.search(pattern, download_url)
+        asset_id = re.search(pattern, download_url).group(1)
         assert asset_id
-        return asset_id.group(1)
+        logger.debug("Found asset ID %s in download URL %s" % (asset_id, download_url))
+        return asset_id
     else:
+        logger.warning(
+            "Download URL doesn't start with tile.loc.gov: %s" % download_url
+        )
         return download_url
 
 
+def do_bagit_export(assets, export_base_dir, export_filename_base):
+    """
+    Executes bagit.py to turn temp directory into LC-specific bagit strucutre.
+    Builds and exports bagit structure as zip.
+    Uploads zip to S3 if configured.
+    """
+
+    for asset in assets:
+        asset_id = get_original_asset_id(asset.download_url)
+        logger.debug("Exporting asset %s into %s" % (asset_id, export_base_dir))
```

Comment:
More deferred string interpolation:

```suggestion
        logger.debug("Exporting asset %s into %s", asset_id, export_base_dir)
```